.travis.yml

```diff
@@ -0,0 +1,30 @@
+language: python
+python:
+  - 3.4
+  - 3.5
+  - 3.6
+
+cache:
+  pip: true
+
+before_install:
+  # Use pip 9, so that setuptools can be downgraded.
+  - pip install pip==9 setuptools==17
```

Comment:
I used `pip install setuptools -U` instead and it worked.

Link: https://travis-ci.org/satwikkansal/coala-quickstart/builds/242965536